tools/pipelines/build-docs.yml

```diff
@@ -22,13 +22,6 @@ parameters:
       - default
```

Comment:
Let's be sure to update the top-level docs explaining what this pipeline does, which has changed.